testinfra/modules/interface.py

```diff
@@ -49,30 +59,74 @@ def get_module_class(cls, host):
             return BSDInterface
         raise NotImplementedError
 
+    @classmethod
+    def names(cls):
+        """Return the names of all the interfaces.
+
+        >>> host.interface.names()
+        ['lo', 'tunl0', 'ip6tnl0', 'eth0']
+        """
+        raise NotImplementedError
+
+    @classmethod
+    def default(cls, family=None):
+        """Return the default interface.
+
```

Comment:
Maybe explain here "interface used for the default route".

Reply:
Ok